### DISCUSSION TITLE
Trying installing on Windows, fails with not finding rich

### DISCUSSION THREAD

**Discussion by ghost** [vladmandic/sdnext#72] [Q&A]
Probably the wrong place so I apologise in advance.
I wanted to give this version a go but having issues with getting it started as it is failing to pick up that I already have rich installed but when trying to run the bat file, it says:
`ModuleNotFoundError: No module named 'rich'`

pip install rich says:
`Requirement already satisfied: rich in c:\users\mine\appdata\local\programs\python\python310\lib\site-packages (13.3.3)`

Any suggestions or am I better just to stick with automatics version?

**Comment by vladmandic** [comment]
Can you provide a longer stack trace, I want to nail down where it's coming from.

**Comment by Centurion-Rome** [comment]
I have manualy installed some missing modules. I don't remember exactly but I had to do something like:
```
cd venv/scripts
.\activate
python -m pip install rich
python -m pip install kornia
python -m pip install piexif
python -m pip install rembg
pip install  -r requirements.txt pip -r requirements_versions.txt

```
you may also copy your "webui-user.bat" from A1111 :)

hope it helps ;)

I have also added 
```
--disable-sdp-attention --xformers
```
to get xformers back ;) 

PS. I moved here because of https://github.com/AUTOMATIC1111/stable-diffusion-webui/discussions/9505

**Comment by vladmandic** [comment]
major update just posted, see #99 for details.
